index.js

```diff
@@ -0,0 +1,87 @@
+$(document).ready(() => {
+
+  const loadTrips = function loadTrips() {
+    $.get(
+      'https://trektravel.herokuapp.com/trips',
+      (response) => {
+        response.forEach((trip) => {
+          const tripsInfo = `<li id="trip-${trip.id}"><p id="${trip.id}">${trip.name}</p></li>`;
+          $('#trips ul').append(tripsInfo);
+        });
+      },
+    )
+      .fail(() => {
+        $('#fail').html('<p>Request was unsuccessful</p>');
+      }).always(() => {
+        console.log('Function: loadTrips (plural)');
+      });
+  }; // end of loadTrips
+
+  const loadTrip = function loadTrip(id) {
+    $.get(`https://trektravel.herokuapp.com/trips/${id}`,
+      (response) => {
+        const tripInfo =
+        `<div>
+          <p> Trip name: ${response.name}</p>
+          <p> Category: ${response.category}</p>
+          <p> Continent: ${response.continent}</p>
+          <p> Description: ${response.about}</p>
+          <p> Duration: ${response.weeks} weeks</p>
+          <p> Cost: ${response.cost}</p>
+        </div>`;
+
+        const reservation = `<form id="form" action="https://trektravel.herokuapp.com/trips/${id}/reservations" method="post">
+            <section>
+              <label>Name</label>
+              <input type="text" id="name" name="name"></input>
+            </section>
+
+            <section>
+              <label>Email</label>
+              <input type="text" id="email" name="email"></input>
+            </section>
+
+            <section class="button">
+              <button type="submit">Reserve Trip</button>
+            </section>
+          </form>`;
+
+        $('#trips ul li > div').hide();
+        $(`#trips ul li#trip-${id}`).append(tripInfo);
+        $(`#trips ul li#trip-${id}`).append(reservation);
+      },
+    ).fail(() => {
+      $('#fail').html('<p>Could not find</p>');
+    }).always(() => {
+      console.log('Function: loadTrip (single)');
+    });
+  }; // end of loadTrip
+
+  $('#trips ul').on('click', 'p', function() {
```

Comment:
You should name this event handler:
`  $('#trips ul').on('click', 'p', function loadTripClickHandler() {`